cmd/root.go

```diff
@@ -57,6 +57,9 @@ func init() {
 	flags.IntP("port", "p", 8080, "server port")
 	viper.BindPFlag("port", flags.Lookup("port"))
 
+	flags.String("fs-url", "files://localhost/tmp/cozy2", "filesystem url")
```

Comment:
Maybe we can use a path relative to the current directory (`./storage`) instead of using `/tmp/cozy2`